Formula/cfonts.rb

```diff
@@ -0,0 +1,25 @@
+class Cfonts < Formula
```

Comment:
Shouldn't we indicate that this is the Rust implementation of `cfonts`?

Reply:
Hey, I'm the author and I feel like node isn't a great way to distribute cli tools which is why I wrote the rust port. So there will never be a nodejs implementation distributes anywhere other than yarn and npm. Does that help?

Reply:
We distribute just over a hundred CLI tools that are installed via `npm`, so I'm not sure we can guarantee that there will never be a distribution of the nodejs implementation other than through `yarn` or `npm`.

Reply:
If someone was to distribute my nodejs library through brew against my consent than at least it should be called `cfonts-nodejs` or something no?

Reply:
The way I phrased my first comment was a bit confusing I admit. I am not saying brew is a bad way to distribute node scripts. I love brew and use it daily. What I meant was that the nodejs implementation of `cfonts` is not as good as the rust one and I would rather publish rust on here than the nodejs one. Hope that's more clear

Reply:
@carlocab Is there anything I can do to unblock this PR?